CIScripts/Test/Tests/vRouterAgentMSIInstaller.Tests.ps1

```diff
@@ -0,0 +1,32 @@
+Param (
+    [Parameter(Mandatory=$true)] [string] $TestbedAddr
+)
+
+. $PSScriptRoot\..\Utils\CommonTestCode.ps1
+. $PSScriptRoot\..\TestConfigurationUtils.ps1
+. $PSScriptRoot\..\..\Common\Aliases.ps1
+. $PSScriptRoot\..\..\Common\VMUtils.ps1
+
+. $PSScriptRoot\..\GetTestConfigurationCodiLegacy.ps1
+$TestConf = Get-TestConfiguration
+$Session = New-PSSession -ComputerName $TestbedAddr -Credential (Get-VMCreds)
+
+Describe "vRouter Agent MSI installer" {
+    Context "installation and uninstallation" {
+        It "registers/unregisters agent service and never enables" {
+            Install-Agent -Session $Session
+            Assert-IsAgentServiceRegistered -Session $Session
+            Assert-IsAgentServiceDisabled -Session $Session
+            Uninstall-Agent -Session $Session
+            Assert-IsAgentServiceUnregistered -Session $Session
```

Comment:
In my personal opinion opinion we should rather use pester's " | should be something", since we are trying to verify whether pester is correct solution for us.